Mage/src/main/java/mage/filter/StaticFilters.java

```diff
@@ -407,6 +407,14 @@ public final class StaticFilters {
         FILTER_CONTROLLED_ARTIFACT_OR_OTHER_CREATURE.setLockedFilter(true);
     }
 
+    public static final FilterControlledPermanent FILTER_CONTROLLED_ANOTHER_ARTIFACT = new FilterControlledPermanent("another artifact");
```

Comment:
The name text should be "another artifact you control"

Reply:
done